userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
+          "reportPrettyName": "New accounts created (per day)",
+          "sourceTableName": "new_accounts_per_day",
+          "refreshProcedureName": "refresh_new_accounts_per_day",
+          "refreshFrequency": "DAILY",
+          "refreshHourOfDayGmt": 4}' \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/reports"
+----
++
+This creates a report called `new_accounts_per_day` based on the `new_accounts_per_day` table. The `refresh_new_accounts_per_day` stored procedure refreshes the table from the view. It runs everyday at 4 AM GMT. Refer to the <<Report Creation Options,Report Creation Options>> section for a description of all the fields.
++
+You can alternatively create the report with a different refresh schedule as follows:
++
+[source,bash]
+----
+curl -v \
+     -X POST \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     -H 'Content-Type: application/json' \
+     -d '{"reportName": "new_accounts_per_day",
+          "reportPrettyName": "New accounts created (per day)",
+          "sourceTableName": "new_accounts_per_day",
+          "refreshProcedureName": "refresh_new_accounts_per_day",
+          "refreshFrequency": "HOURLY"}' \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/reports"
+----
++
+This specifies that the stored procedure should run on an hourly basis (it will run at 5' past the hour)
+
+[[advanced_report_creation_options]]
+=== Advanced Report Creation Options
+
+You can create a report based on SQL executed on a remote database (`warehouse`) as follows:
+
+[source,bash]
+----
+curl -v \
+     -X POST \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     -H 'Content-Type: application/json' \
+     -d '{"reportName": "report_historical_orders",
+          "reportType": "TABLE",
+          "reportPrettyName": "Historical orders",
+          "sourceName": "warehouse",
+          "sourceQuery": "select * from warehouse.public.orders"}' \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/reports"
+----
+
+This creates a report called `report_historical_orders` based on a remote database (`warehouse`). It executes the specified SQL query. Refer to the <<Report Creation Options,Report Creation Options>> section for a description of all the fields.
+
+[NOTE]
+*Note:* In order to create a report based on a remote database, the database first needs to be configured at a tenant level as explained in the <<remote_database_configuration,plugin configuration>> section. 
+
+The above `cURL` command can also be used to create a report based on a local database by directly specifying an SQL query (instead of a table/view).
+
+=== Report Creation Options
+
+The following table lists all the report creation options.A subset of these options need to be specified while <<create_custom_reports,creating the report>> depending on the report being created.
+
+[options="header",cols="1,1"]
+|===
```

Comment:
Here I would again use periods at the end of sentences to maintain consistency.

Reply:
Fixed